backend-ci/tests/_support/Fakes/FakeWebhookSubscriptionRepository.php

```
@@ -0,0 +1,60 @@
+        $this->rows[] = $row;
+        return $row;
+    }
+
+    public function activeForEvent(string $event): array
+    {
+        return array_values(array_filter($this->rows, fn ($r) =>
+            ($r['event'] ?? '') === $event && ($r['is_active'] ?? 1)
+        ));
+    }
+
+    public function findAll(array $filters = []): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$filters'. (undefined)

(UnusedFormalParameter)